### ISSUE TITLE
Truffle-contract docs outdated

### ISSUE THREAD

**Issue by wbt** [trufflesuite/truffle#1588]
---------------------------

## Issue

In the Truffle v5 documentation, "[Interacting with your contracts](https://truffleframework.com/docs/truffle/getting-started/interacting-with-your-contracts#introducing-abstractions)" describes and links to the truffle-contract module.  The readme at the link is outdated.  

With the time invested so far, I cannot tell exactly how much of the readme is outdated, but I can see that the Full Example links to the deprecated [truffle-artifactor repo](https://github.com/trufflesuite/truffle-artifactor), and the [current readme](https://github.com/trufflesuite/truffle/tree/develop/packages/truffle-artifactor)'s backlink for the truffle-contract package goes to a similarly deprecated repo.

With the release of v5, it would be nice to review the readme files of these packages to see how much needs to be changed/updated, as the current state of documentation post-release is very confusing.  A user does not know what information is still accurate and what is deprecated with the new version, without extra hours of effort experimenting.

Related: Issue #1587 about version number mismatch.

**Comment by eggplantzzz** [comment]
@wbt Thanks for bringing this to our attention. We are currently trying to work to get everything up to date and consistent!

**Comment by aakilfernandes** [comment]
Also confused by this

**Comment by wbt** [comment]
While the specific links cited above have been addressed, I think the documentation is still generally outdated and would prefer not to see the issue autoclosed until some deliberate effort has been put into reviewing and updating documentation.

**Comment by adrianmcli** [comment]
@wbt I agree, perhaps you can start on a list of things you believe should be fixed?

**Comment by wbt** [comment]
Thanks for the vote of confidence, though I'm pretty stacked up with other tasks at the moment (in part due to lack of documentation) and don't have time to do that review at present.

**Comment by eggplantzzz** [comment]
Hmm I did make some updates that resolved a lot of those issues I think. When I have some time I'll try and see how much of the stuff is still old. So rest assured, there was some effort to remedy this :)
